airgun/views/ansible_variable.py

```diff
@@ -17,9 +17,9 @@
 class AnsibleVariablesView(BaseLoggedInView, SearchableViewMixin):
     """Main Ansible Variables view"""
 
-    title = Text("//h1[contains(., text()='Ansible Variables')")
+    title = Text("//h1")
```

Comment:
This works now but can we add  "Text("//h1[contains(normalize-space(.)='Ansible Variables')")". What do you say @omkarkhatavkar ?

Reply:
Yes, we can us the normalised space, the earlier locator itself was wrong. @shweta83 you shared also needs to correct. It should be `"//h1[contains(normalize-space(.),'Ansible Variable')]"`

Reply:
Thanks @omkarkhatavkar.

Reply:
Updated @omkarkhatavkar @shweta83